### PR TITLE
Hi] localize concepts/architecture/leases into hindi

### PR DESCRIPTION
**Pull request by KumareshSingha97**
Hello

I have changed docs/concepts/architecture/leases page English language to Hindi

### CODE REVIEW

**Comment by NitishKumar06** [comment]
Couple of things to do before passing this PR for a review:

1. Sign the CLA
2. Please mention the link to the page that is being updated.

Putting this PR on hold. Feel free to unhold once you're done signing the CLA.
/hold
/assign @bishal7679

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,1317 @@
...
+    "node_modules/yargs-parser/node_modules/decamelize": {
+      "version": "1.2.0",
+      "resolved": "https://registry.npmjs.org/decamelize/-/decamelize-1.2.0.tgz",
+      "integrity": "sha1-9lNNFRSCabIDUue+4m9QH5oZEpA=",
+      "dev": true,
+      "engines": {
+        "node": ">=0.10.0"
+      }
+    }
+  },
```
Comment (NitishKumar06) [inline package-lock.json:1314]: Not sure why are these changes being made. @bishal7679 I don't think we had made changes to the file `package-lock.json` for any localization issue/PR. Even if we did, the file would have been in `.gitignore`.

**Review by NitishKumar06 — COMMENTED** [review]
PTAL!

File: package-lock.json (hunk 1 above)
Comment (KumareshSingha97) [reply package-lock.json:1314]: Hi @NitishKumar06 sir, i have resolved the issue on `package-lock.json` .Shall i again create a pull request?

Comment (NitishKumar06) [reply package-lock.json:1314]: Link the issue this PR is solving in the PR description & Sign the CLA. 

Also, I'd recommend you go through the contributing docs.

**Review by Arhell — COMMENTED** [review]
/easycla

**Review by KumareshSingha97 — COMMENTED** [review]
/easycla

**Pull request closed without merging**